### DISCUSSION TITLE
Removing angularJS from umbraco dashboard

### DISCUSSION THREAD

**Discussion by niknam-mohsen** [umbraco/Umbraco-CMS#15122] [Features and ideas]
When will angularjs be removed from the umbraco dashboard? is there a plan for this?

**Comment by bergmania** [comment]
Plan is all remaining angularjs will be removed in Umbraco 14. Previews are available, but this will be massively breaking.

**Reply by niknam-mohsen** [reply]
Thank you for your reply.
When will the final version 14 be released?

**Reply by leekelleher** [reply]
Umbraco 14 is scheduled for release on May 30, 2024.
For reference: <https://umbraco.com/products/knowledge-center/long-term-support-and-end-of-life/>
